components/filter/filter-overflow-group.js

```diff
@@ -27,16 +40,46 @@ class FilterOverflowGroup extends OverflowGroupMixin(RtlMixin(LitElement)) {
 		`];
 	}
 
+	constructor() {
+		super();
+
+		this._filterIds = '';
+	}
+
 	firstUpdated(changedProperties) {
 		super.firstUpdated(changedProperties);
 
 		this.addEventListener('d2l-filter-change', this._handleFilterChange);
+
+		if (!this.tags) return;
+
+		this._filterTags = document.createElement('d2l-filter-tags');
+		this.insertAdjacentElement('afterend', this._filterTags);
```

Comment:
This adds a new `d2l-filter-tags` element as a sibling of the overflow group so that it can access/subscribe to the slotted filters.

If styles need to be changed, it will need to be done here rather than in `styles` like usual

Reply:
Would it work if we made this a child instead, so its still in the light dom but gets cleaned up if the filter overflow group is `diconnected`?  I guess that'd probably mess with all the chomping 🤔

Reply:
I think I had tried that and it gets pulled into the overflow slot. Ah yeah should this be added/deleted in the connected/disconnected callback?

Reply:
Yeah I think `connected`/`disconnected` should work.  Was trying to think if we could do anything with a new non-default `slot`, but even so that might be weird since it'd be available to consumers but we wouldn't want them to use it

Reply:
@dlockhart @dbatiste Wondering if you have any major concerns here I've missed... since this mainly a convenience feature, I just want to make sure we're not setting ourselves up for future headaches with supporting this.
This allows the consumer to just add the `tags` attribute to this component and it will add the `d2l-filter-tags` after itself and hookup the ids for them, rather than having to do it like the "Filter Overflow Group with Tags (external to d2l-filter-overflow-group)" demo above.

Reply:
Hmm yeah I don't love this for a couple reasons -- we're modifying the DOM outside of `render()` and the component is essentially "leaking" a sibling, which is pretty unexpected. That shows up in places like the unit and visual diff tests where we had to wrap it in a seemingly unnecessary `<div>` in order to properly capture both.

Could the overflow-group mixin expose another optional method like `getOverflowAdjacentElement()` (or a better name) that it would render after its `<div class="d2l-overflow-group-container">`?

Reply:
Yeah, I can play around with that. I like that better than trying to do something similar with a new `slot`.

Reply:
> Could the overflow-group mixin expose another optional method like getOverflowAdjacentElement() (or a better name) that it would render after its <div class="d2l-overflow-group-container">?

So turns out if we do this, then the `d2l-filter-tags` will be in the `d2l-filter-overflow-group`'s shadow DOM, but the `d2l-filter`s are all in the light DOM.  So they can't see each other.  I'm not seeing a way to get the tags in the light DOM other than what we do here (attaching them after `this`) or having the consumer place them in a `slot` (which is basically the same as having no `slot` at all and the consumers handling it).  If we add them as children, they get handled with all the other chomping calculations.

Reply:
> or having the consumer place them in a slot (which is basically the same as having no slot at all and the consumers handling it)

What about having a "silent" slot - it's there, the consumers aren't expected to use it, and the component injects the `d2l-filter-tags` into the slot, instead of as a child or sibling?

Reply:
Originally I was trying to avoid the slot but it's actually a lot cleaner than I expected! I'll put it up as a branched PR to make it easier to see that change, thanks!